src/sentry/migrations/0121_obliterate_group_inbox.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-11-02 20:25
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+BATCH_SIZE=100
+
+def obliterate_group_inbox(apps, schema_editor):
```

Comment:
There's a utility function for bulk deletion.

https://github.com/getsentry/sentry/blob/4847c3c84403840621e6962991346f3663fa4046/src/sentry/utils/query.py#L147-L192

It'll be so much faster because the whole thing happens in the database, without the need to create python objects for `GroupInbox` rows.

Reply:
We're not bringing the rows into memory here either, this all runs as one query with a subquery.